src/js/lib/content/src/index.js

```diff
@@ -121,6 +121,22 @@ function show_debug_screen(wasm,msg) {
 }
 
 
+
+// =======================================
+// === Copy Documentation Example Code ===
+// =======================================
+
+function copyCode(codeBlockElem) {
+    let range = document.createRange()
+    range.selectNode(codeBlockElem)
+    window.getSelection().removeAllRanges()
+    window.getSelection().addRange(range)
+    document.execCommand('copy')
```

Comment:
This has a few problems:
1. It is synchronious, so when used for the first time, it can display a message that a script wants to access the clipboard and it will freeze the whole GUI until someone clicks "allow" or "deny".
2. We have developed good copy handling in Rust already, which uses asynchronious callbacks and fall backs to `execCommand` in case these are not supported. Please use these instead.